components/patterns/lockup.tsx

```diff
@@ -1,19 +1,26 @@
 import Link from "next/link";
+import {ComponentProps} from "react";
 
 import Logo from "@/components/simple/logo";
 
-export const Lockup = (props) => {
+interface LockupProps extends ComponentProps<any> {
+  whiteText?: boolean
+}
+
+export const Lockup = ({whiteText, ...props}: LockupProps) => {
   return (
     <div {...props}>
       <Link href="/" className="su-no-underline">
         <div className="su-flex md:su-flex-row su-basefont-19">
-          <div><Logo /></div>
+          <Logo/>
 
           <div className="su-flex su-flex-wrap lg:su-items-center su-ml-10">
-            <div className="lg:su-flex su-items-baseline su-leading-none su-w-full lg:su-w-auto su-mt-15 lg:su-mt-0 su-logo su-text-cardinal-red su-type-4 su-mr-5 su-pr-5 lg:su-border-r su-border-solid su-border-black">
+            <div
+              className={"lg:su-flex su-items-baseline su-leading-none su-w-full lg:su-w-auto su-mt-15 lg:su-mt-0 su-logo su-type-4 su-mr-5 su-pr-5 lg:su-border-r su-border-solid su-border-black" + (whiteText ? " su-text-white" : " su-text-cardinal-red")}>
```

Comment:
- Lockup border right is pretty hard to see in the footer. You may want to consider lightening that up a bit.
- Can you add `hover:su-text-white` if `whiteText` as well? Currently red on hover in the footer.

Reply:
Cool call